src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -70,6 +70,19 @@ public SwerveDriveMap getSwerveDriveMap() {
                 maxRotationRadianPerSecond, gyro);
     }
 
+    @Override
+    public ShooterMap getShooterMap() {
+        final PIDSparkMax shooterMoter = new PIDSparkMax(13, MotorType.kBrushless);
+        final PIDSparkMax shooterLoadingMoter = new PIDSparkMax(14, MotorType.kBrushless);
+        final double shootWheelRadius = 2.0; // Radius in inches
+
+        // Convert to rotations per minute?
+        final double velocityMultiplier = shootWheelRadius * 2 * Math.PI / (60 * 12);
```

Comment:
- Where does the 12 come from?
- I suggest using `setPositionScaleFactor()` and `setVelocityScaleFactor()` for this.